README.md

```diff
@@ -89,6 +89,55 @@ RUN pip install --no-cache-dir <A_REQUIRED_PACKAGE_ON_PYPI>
 USER 1001
 ```
 
+### Add Sanic extensions
+With the use of Pluggy, you can now create additional sanic extensions by accessing the app object created by the action server.
+
+#### Step by Step Guide on creating your own sanic extension in rasa_sdk
+This example will show how to create a sanic listener using plugins
+
+##### Step 1
+Create a package in your action server project called `rasa_sdk_plugins`. Rasa SDK will try to instantiate this package in your project to start plugins.
+If no plugins found, it will print an info that there are no plugins in your project.
```

Comment:
I didn't see any logger info message in cases of `ModuleNotFound`, perhaps best to add that in the sdk code?

Reply:
yup, i wrote the docs and removed the log messages 😮‍💨 i will add it back